bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -766,10 +766,7 @@ private String getStatusMessage(PullRequestCheckIssueVisitor visitor,
                                         "a new backport will be created when this pr is integrated.)");
                             }
                         }
-                        var titleIssue = Issue.fromStringRelaxed(pr.title());
-                        var isTitleIssue = titleIssue.isPresent() && titleIssue.get().shortId().equals(issue.shortId());
-                        // Only check title mismatch for title issue
-                        if (isTitleIssue && !relaxedEquals(issueTrackerIssue.get().title(), issue.description())) {
+                        if (!relaxedEquals(issueTrackerIssue.get().title(), issue.description())) {
```

Comment:
I reverted SKARA-2253 here. Since the issue marker will be updated automatically, the title in the issue marker should be equal with the title of issuetrackerissue. If they are not equal, it means some issues happened, so we'd better add a warning here.